graph-http/src/intoresponse.rs

```diff
@@ -167,9 +174,9 @@ where
 }
 
 impl<'a, T> IntoResponseBlocking<'a, T> {
-    pub fn json<U>(self) -> GraphResult<GraphResponse<U>>
+    pub fn json<U, V>(self) -> GraphResult<GraphResponse<U>>
```

Comment:
Good job with the README comments. I was more referring to a method comment though.